### ISSUE TITLE
Fix dependencies

### ISSUE THREAD

**Issue by genryxy** [artipie/composer-adapter#116]
Now, Composer has the next tree of dependencies
```
[INFO] com.artipie:composer-adapter:jar:1.0
[INFO] +- com.google.guava:guava:jar:21.0-rc2:compile
[INFO] +- com.artipie:asto:jar:v1.3.4:compile
[INFO] |  +- com.jcabi:jcabi-log:jar:0.19.0:compile
[INFO] |  +- com.github.akarnokd:rxjava2-jdk8-interop:jar:0.3.7:compile
[INFO] |  |  \- io.reactivex.rxjava2:rxjava:jar:2.2.10:compile
[INFO] |  +- com.github.akarnokd:rxjava2-extensions:jar:0.20.10:compile
[INFO] |  |  \- org.reactivestreams:reactive-streams:jar:1.0.3:compile
[INFO] |  +- org.cqfn:rio:jar:0.3:compile
[INFO] |  |  \- org.jctools:jctools-core:jar:3.1.0:compile
[INFO] |  +- io.github.resilience4j:resilience4j-retry:jar:1.5.0:compile
[INFO] |  |  +- io.vavr:vavr:jar:0.10.2:compile
[INFO] |  |  |  \- io.vavr:vavr-match:jar:0.10.2:compile
[INFO] |  |  \- io.github.resilience4j:resilience4j-core:jar:1.5.0:compile
[INFO] |  \- commons-codec:commons-codec:jar:1.12:compile
[INFO] +- javax.json:javax.json-api:jar:1.1.4:compile
[INFO] +- com.artipie:http:jar:v0.19:compile
[INFO] |  +- org.apache.commons:commons-lang3:jar:3.8:compile
[INFO] |  +- org.apache.commons:commons-collections4:jar:4.3:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.13:compile
[INFO] |  |  +- org.apache.httpcomponents:httpcore:jar:4.4.13:compile
[INFO] |  |  \- commons-logging:commons-logging:jar:1.2:compile
[INFO] |  \- wtf.g4s8:mime:jar:2.0.0:compile
[INFO] +- com.artipie:http-client:jar:0.3.6:compile
[INFO] |  \- org.eclipse.jetty:jetty-reactive-httpclient:jar:1.1.10:compile
[INFO] |     \- org.eclipse.jetty:jetty-client:jar:9.4.43.v20210629:compile
[INFO] |        +- org.eclipse.jetty:jetty-http:jar:9.4.43.v20210629:compile
[INFO] |        |  \- org.eclipse.jetty:jetty-util:jar:9.4.43.v20210629:compile
[INFO] |        \- org.eclipse.jetty:jetty-io:jar:9.4.43.v20210629:compile
[INFO] +- org.apache.commons:commons-compress:jar:1.21:compile
[INFO] +- org.junit.jupiter:junit-jupiter-api:jar:5.7.1:test
[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.0:test
[INFO] |  +- org.opentest4j:opentest4j:jar:1.2.0:test
[INFO] |  \- org.junit.platform:junit-platform-commons:jar:1.7.1:test
[INFO] +- org.junit.jupiter:junit-jupiter-engine:jar:5.7.1:test
[INFO] |  \- org.junit.platform:junit-platform-engine:jar:1.7.1:test
[INFO] +- org.hamcrest:hamcrest:jar:2.2:test
[INFO] +- org.glassfish:javax.json:jar:1.1.4:test
[INFO] +- org.cactoos:cactoos:jar:0.46:test
[INFO] +- org.slf4j:slf4j-log4j12:jar:1.8.0-beta1:test
[INFO] |  \- org.slf4j:slf4j-api:jar:1.8.0-beta1:compile
[INFO] +- log4j:log4j:jar:1.2.17:test
[INFO] +- com.artipie:vertx-server:jar:0.3.2:test
[INFO] |  \- io.vertx:vertx-rx-java2:jar:3.9.2:test
[INFO] |     +- io.vertx:vertx-core:jar:3.9.2:test
[INFO] |     |  +- io.netty:netty-common:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-buffer:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-transport:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-handler:jar:4.1.49.Final:test
[INFO] |     |  |  \- io.netty:netty-codec:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-handler-proxy:jar:4.1.49.Final:test
[INFO] |     |  |  \- io.netty:netty-codec-socks:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-codec-http:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-codec-http2:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-resolver:jar:4.1.49.Final:test
[INFO] |     |  +- io.netty:netty-resolver-dns:jar:4.1.49.Final:test
[INFO] |     |  |  \- io.netty:netty-codec-dns:jar:4.1.49.Final:test
[INFO] |     |  +- com.fasterxml.jackson.core:jackson-core:jar:2.10.2:test
[INFO] |     |  \- com.fasterxml.jackson.core:jackson-databind:jar:2.10.2:test
[INFO] |     \- io.vertx:vertx-rx-gen:jar:3.9.2:test
[INFO] |        \- io.vertx:vertx-codegen:jar:3.9.2:test
[INFO] |           \- org.mvel:mvel2:jar:2.3.1.Final:test
[INFO] +- com.artipie:files-adapter:jar:0.7:test
[INFO] +- org.junit.jupiter:junit-jupiter-params:jar:5.7.1:test
[INFO] +- org.testcontainers:testcontainers:jar:1.16.0:test
[INFO] |  +- junit:junit:jar:4.12:test
[INFO] |  |  \- org.hamcrest:hamcrest-core:jar:1.3:test
[INFO] |  +- org.rnorth.duct-tape:duct-tape:jar:1.0.8:test
[INFO] |  +- com.github.docker-java:docker-java-api:jar:3.2.11:test
[INFO] |  |  \- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.2:test
[INFO] |  \- com.github.docker-java:docker-java-transport-zerodep:jar:3.2.11:test
[INFO] |     +- com.github.docker-java:docker-java-transport:jar:3.2.11:test
[INFO] |     \- net.java.dev.jna:jna:jar:5.8.0:test
[INFO] +- org.testcontainers:junit-jupiter:jar:1.16.0:test
[INFO] +- com.google.code.findbugs:annotations:jar:3.0.1:provided
[INFO] \- org.mockito:mockito-core:jar:3.5.0:test
[INFO]    +- net.bytebuddy:byte-buddy:jar:1.10.13:test
[INFO]    +- net.bytebuddy:byte-buddy-agent:jar:1.10.13:test
[INFO]    \- org.objenesis:objenesis:jar:3.1:test
```
Problems:
- **high-risk vulnerabilities**
  - `log4j/log4j/1.2.17` --- (test dependency)
  - `com.fasterxml.jackson.core/jackson-databind/2.10.2` --- (test dependency)
  - `org.apache.commons/commons-compress/1.20` --- (bumped to 1.21)
  - `org.eclipse.jetty/jetty-io/9.4.30.v20200611` --- (http-client:jar was bumped to 0.3.6 therefore the version of jetty-client is 9.4.43)
  - `io.vertx/vertx-core/3.9.2` --- (test dependency) 
It is necessary to fix these problems by excluding / removing, updating dependencies

**Comment by genryxy** [comment]
@g4s8 could you please release a new version

**Comment by g4s8** [comment]
@genryxy please push a tag to repo with a new version and changelog, you have "maintainer" permissions to do that.

**Comment by genryxy** [comment]
Closed as problems with high-risk vulnerabilities were solved and a new version [`v1.0.1`](https://github.com/artipie/composer-adapter/releases/tag/v1.0.1) was released.
